app/views/general_shopping_lists/general.html.erb

```diff
@@ -0,0 +1,28 @@
+<section class="container flex food-placeholder">
+<div class="food flex">
+  <h3 class="title">General Shopping List</h3>
+  <div class="sum flex">
+    <p>Amount of food to buy: <%= @general_shopping_lists.count %> </p>
+    <p>Total value of food needed: $<%= @general_shopping_lists.map { |food| food.price}.sum %></p>
+  </div>
+
+  <table>
+    <thead>
+      <tr>
+        <th>Food</th>
+        <th>Quantity</th>
+        <th>Price</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @general_shopping_lists.each do |food| %>
+        <tr>
+          <td><%= food.name %></td>
+          <td><%= food.quantity %> <%= food.measurement_unit %></td>
+          <td>$<%= food.price %></td>
+        </tr>
+      <% end %>
+    </tbody>
+  </table>
+</div>
+</section>
```

Comment:
- [ ] Kindly implement the general shopping list correctly as it supposes to show only the missing foods for all recipes of the logged-in user as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md#:~:text=Should%20show%20the%20list%20of%20food%20that%20is%20missing%20for%20all%20recipes%20of%20the%20logged%2Din%20user%20(compare%20the%20list%20of%20food%20for%20all%20recipes%20with%20the%20general%20food%20list%20of%20that%20user).).